src/routes/products/search/+page.svelte

```diff
@@ -32,7 +32,7 @@
 		if (!result?.hits || result.hits.length === 0 || !data.attributesByCode) return [];
 
 		const productsWithAttributes = result.hits.map((product) => ({
-			...product,
+			product: product,
```

Comment:
```suggestion
			product: hit.product,
```

and rename `(product)` to `(hit)`. Then remove the `product.product` from the rest of the file

Reply:
I think we can do this, it will be much better.
<img width="679" height="267" alt="Screenshot 2025-08-22 at 6 45 49 PM" src="https://github.com/user-attachments/assets/29d3b9cf-9ce9-4276-8f2d-37667b7f91c1" />